org.freeorion.FreeOrion.json

```diff
@@ -0,0 +1,60 @@
+{
+  "id": "org.freeorion.FreeOrion",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "freeorion",
+  "rename-icon": "freeorion",
+  "rename-desktop-file": "freeorion.desktop",
+  "rename-appdata-file": "freeorion.appdata.xml",
+  "finish-args": [
+    "--socket=x11",
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--share=network"
+  ],
+  "modules": [
+    "shared-modules/glu/glu-9.0.0.json",
+    "shared-modules/glew/glew.json",
+    {
+      "name": "boost",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.bz2",
+          "sha256": "7f6130bc3cf65f56a618888ce9d5ea704fa10b462be126ad053e80e553d6d8b7"
+        }
+      ],
+      "build-commands": [
+        "./bootstrap.sh --prefix=${FLATPAK_DEST}",
+        "./b2 headers",
+        "./b2 install"
+      ],
+      "cleanup": [
+        "/include"
+      ]
+    },
+    {
+      "name": "freeorion",
+      "buildsystem": "cmake",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/freeorion/freeorion/archive/v0.4.8.zip",
+          "sha256": "aa25cb4333a01dd6f554183602f7c84de8934a2d4e8651c64c7dfcdd60a4b3d6"
+        },
+        {
+          "type": "file",
+          "path": "freeorion.appdata.xml"
+        }
+      ],
+      "post-install": [
+        "mkdir -p /app/share/appdata",
```

Comment:
Just use install instead. And you can also install the appdata file with the correct filename and remove the rename-appdata-file from the manifest. The file must be named `org.freeorion.FreeOrion.appdata.xml` and should be installed under `/app/share/metainfo` (it's the new location that should be used instead of /appdata/)